src/extension.js

```diff
@@ -168,7 +153,7 @@ const ServiceIndicator = GObject.registerClass({
         const panelMode = this.settings.get_boolean('show-indicators');
 
         // Hide status indicator if in Panel mode or no devices are available
-        this._indicator.visible = (!panelMode && available.length);
+        featureIndicator._indicator.visible = (!panelMode && available.length);
```

Comment:
```suggestion
        serviceIndicator._indicator.visible = (!panelMode && available.length);
```

Renaming for consistency (ServiceIndicator, ServiceToggle).